stable/datadog/values.yaml

```diff
@@ -1,290 +1,256 @@
-  # volumes:
-  #  - hostPath:
-  #      path: /host/path
-  #    name: myvolume
-  # volumeMounts:
-  #   - name: myvolume
-  #     mountPath: /container/path
-  #     readOnly: true
-
-  ## Enable leader election mechanism for event collection
-  ##
-  # leaderElection: false
-
-  ## Set the lease time for leader election
-  ##
-  # leaderLeaseDuration: 600
-
+  #
+  #  env:
+  #    - name: <ENV_VAR_NAME>
+  #      value: <ENV_VAR_VALUE>
+
+  ## @param volumes - list of objects - optional
+  ## Specify additional volumes to mount in the dd-agent container
+  #
+  #  volumes:
+  #    - hostPath:
+  #      path: <HOST_PATH>
+  #      name: <VOLUME_NAME>
+
+  ## @param volumeMounts - list of objects - optional
+  ## Specify additional volumes to mount in the dd-agent container
+  #
+  #  volumeMounts:
+  #    - name: <VOLUME_NAME>
+  #      mountPath: <CONTAINER_PATH>
+  #      readOnly: true
+
+  ## @param confd - list of objects - optional
   ## Provide additional check configurations (static and Autodiscovery)
-  ## Each key will become a file in /conf.d
+  ## Each key becomes a file in /conf.d
   ## ref: https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent#optional-volumes
   ## ref: https://docs.datadoghq.com/agent/autodiscovery/
-  ##
-  # confd:
-  #   redisdb.yaml: |-
-  #     init_config:
-  #     instances:
-  #       - host: "name"
-  #         port: "6379"
-  #   kubernetes_state.yaml: |-
-  #     ad_identifiers:
-  #       - kube-state-metrics
-  #     init_config:
-  #     instances:
-  #       - kube_state_url: http://%%host%%:8080/metrics
-
+  #
+  #  confd:
+  #    redisdb.yaml: |-
+  #      init_config:
+  #      instances:
+  #        - host: "name"
+  #          port: "6379"
+  #    kubernetes_state.yaml: |-
+  #      ad_identifiers:
+  #        - kube-state-metrics
+  #      init_config:
+  #      instances:
+  #        - kube_state_url: http://%%host%%:8080/metrics
+
+  ## @param checksd - list of key:value strings - optional
   ## Provide additional custom checks as python code
-  ## Each key will become a file in /checks.d
+  ## Each key becomes a file in /checks.d
   ## ref: https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent#optional-volumes
-  ##
-  # checksd:
-  #   service.py: |-
+  #
+  #  checksd:
+  #    service.py: |-
 
+  ## @param criSocketPath - string - optional 
   ## Path to the container runtime socket (if different from Docker)
   ## This is supported starting from agent 6.6.0
-  # criSocketPath: /var/run/containerd/containerd.sock
-
-  ## Provide a mapping of Kubernetes Labels to Datadog Tags
-  # podLabelsAsTags:
-  #     app: kube_app
-  #     release: helm_release
-
+  # 
+  #  criSocketPath: /var/run/containerd/containerd.sock
+
+  ## @param podLabelsAsTags - list of key:value strings - optional
+  ## Provide a mapping of Kubernetes Labels to Datadog Tags.
+  #
+  #  podLabelsAsTags:
```

Comment:
podLabelsAsTags and podAnnotationsAsTags are pretty commonly used, do they deserve to be higher in the list?